specification/schemas/shops/Shop.json

```diff
@@ -7,7 +7,6 @@
       "properties": {
         "attributes": {
           "type": "object",
-          "additionalProperties": false,
```

Comment:
Let's leave this additionalProperties: false here, since it protects from sending attributes that are not defined on the schema.

Instead, we can define the `return_portal_url` attribute in the `Shop` schema and make it readOnly (by adding `"readOnly": true` to the property). 

Don't forget to add the `return_portal_url` attribute in the `required` array of the `ShopResponse` schema ;)